adapters/repos/db/index.go

```diff
@@ -753,6 +753,23 @@ func (i *Index) IncomingExists(ctx context.Context, shardName string,
 	return ok, nil
 }
 
+func propertyIsIndexed(schemaDefinition *models.Schema, className, tentativePropertyName string) bool {
+	propertyName := strings.Split(tentativePropertyName, "^")[0]
```

Comment:
This is because properties can have a "boost".  For instance:  "title" can have "title^2", which increases the score for title matches.

So I have to remove the ^2 to get the correct property name

Reply:
ahh, thanks!